src/tenderly/devnet-execute.ts

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable @typescript-eslint/no-var-requires, no-console */
+import { promisify } from "util";
+import { exec } from "child_process";
+
+const execAsync = promisify(exec);
+
+const spawnCommand = "ts-node src/tenderly/spawn-devnet.ts";
+const opCommandBase = "yarn hardhat run";
+const networkArg = "--network devnet";
+const opsPath = "src/tenderly/run-all-flows.ts";
+
+/**
+ * Top level function to execute everything on the DevNet.
+ * It executes 2 child processes:
+ * 1. Spawn a DevNet through the helper and ts-node directly,
+ * this will also set all the required env vars, so that Hardhat can correctly
+ * work with contracts and auth in Tenderly
+ * 2. Launch deploy and operation flow with contracts using Hardhat
+ *
+ * To execute this, uncomment `tenderly.setup()` line in the hardhat.config.ts
+ * then run `yarn devnet` in the terminal.
+ * */
+const execute = async () => {
+  // spawn DevNet on Tenderly with ts-node directly
+  const spawnRes = await execAsync(spawnCommand);
+
+  const opCommand = `${opCommandBase} ${opsPath} ${networkArg}`;
```

Comment:
what's the point of making these 3 different variables when they are all only referenced here, a single time? Just make the command one variable

Reply:
it's more future proofing. so we can change parts of it independently if path or network changes. but I will look into it. maybe it's not necessary